### PR TITLE
adding annotations

### PR DESCRIPTION
**Pull request by tnguyencota**
**What this PR does / why we need it**:
Adding annotations so it can be use by tool such as sysdig to pull metrics

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/prometheus-postgres-exporter/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ The following tables lists the configurable parameters of the postgres Exporter
 | `tolerations`                   | Add tolerations                            | `[]`  |
 | `nodeSelector`                    | node labels for pod assignment | `{}`  |
 | `affinity`                       |     node/pod affinities | `{}` |
+| `annotations`                    | Deployement annotations | `{}` |
```
Comment (gianrubio) [inline stable/prometheus-postgres-exporter/README.md]: There’s a typo on deployment

**Review by gianrubio — COMMENTED** [review]
Just a minor typo

**Comment by gianrubio** [comment]
please fix CI

**Comment by timstoop** [comment]
If you add a `home` key to Chart.yaml (I think it should point to the URL that's also set in `sources`), CI will succeed.

**Comment by timstoop** [comment]
Please also remove the blank line at the end of values.yaml.

**Comment by tnguyencota** [comment]
strange.  it was complaining about no new line at end of file earlier :(
removed the blank line.  thx

**Comment by timstoop** [comment]
It looks like the image is trying to overwrite something on a read-only filesystem?

**Comment by tnguyencota** [comment]
this https://github.com/kubernetes/charts/pull/5473 would probably fixing the build failure
guess i will wait for that to merge into master and rebase my branch with it

if there is a better way of doing this, please let me know.  thx

**Comment by gianrubio** [comment]
Indeed #5473 will fix this issue

**Comment by tnguyencota** [comment]
@gianrubio i rebased and have https://github.com/kubernetes/charts/pull/5473 in my branch.  would you approve so this can move through the queue.  thx

**Pull request merged**